cli/src/pcluster/templates/cdk_builder.py

```diff
@@ -64,3 +70,12 @@ def build_imagebuilder_template(image_config: ImageBuilderConfig, image_id: str,
             generated_template = load_yaml_dict(os.path.join(tempdir, f"{output_file}.template.json"))
 
         return generated_template
+
+    @staticmethod
+    def load_manifest_json(
```

Comment:
is this used?

Reply:
No, it's a remnant from an earlier implementation based on parsing the manifest.json. I'll remove it in the next revision.